src/Rings.jl

```diff
@@ -241,3 +241,40 @@ function is_finite(R::NCRing)
   c == 1 && return true
   throw(NotImplementedError(:is_finite, R))
 end
+
+function _is_noetherian(R::T) where T<:Ring
+  if R isa Union{PolyRing, MPolyRing, LaurentPolyRing, LaurentMPolyRing}
+    return _is_noetherian(coefficient_ring(R))
+  elseif R isa Union{MSeriesRing, SeriesRing}
+    return _is_noetherian(base_ring(R))
+  elseif R isa Union{Integers, <:Field}
+    return true
+  else
+    return false
```

Comment:
We should not do this. It will lead (down the road) to confusing errors of the kind "can't compute Krull dimension because this ring is not Noetherian" but actually the ring is Noetherian and we just don't know it... which will confuse anyone seeing or debugging this.

Instead I'd be happy to just have a "method not found" error, i.e., provide no generic fallback method.